web/packages/core/src/load-ruffle.ts

```diff
@@ -55,8 +61,40 @@ async function fetchRuffle(config: Config): Promise<typeof Ruffle> {
     const { default: init, Ruffle } = await (extensionsSupported
         ? import("../pkg/ruffle_web-wasm_extensions")
         : import("../pkg/ruffle_web"));
+    let response;
+    const wasmUrl = extensionsSupported
+        ? new URL("../pkg/ruffle_web-wasm_extensions_bg.wasm", import.meta.url)
+        : new URL("../pkg/ruffle_web_bg.wasm", import.meta.url);
+    const wasmResponse = await fetch(wasmUrl.href);
```

Comment:
nit: `fetch()` can accept a `URL` as a parameter: https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters. So it should be possible to omit the `.href`.